### DISCUSSION TITLE
youtube ad revenue

### DISCUSSION THREAD

**Discussion by dabbingsteve19** [dhowe/AdNauseam#2316] [Q&A]
since adnauseum clicks ads instead of blocking them, does that mean individuals who rely on ad revenue such as youtubers will get paid with this extension as opposed to adblockers that block the ad in its entirety?

**Comment by mneunomne** [comment]
@dabbingsteve19 In theory yes. But the reality is much more complicated. Here are some of the many points we have to take into consideration:

Not all websites have ads we _can_ click. For example, youtube, facebook, instagram have ad dom structures that are constantly changing, which makes virtually impossible for us to design parsers to collect them. So on those cases, we just block the ads.

On website that we can indeed click on ads, they will have more visit and theoretically be paid for it. But reality is that the Ad Network have sophisticated methods of detecting unusual behaviour of fake-ad-clicks, but we believe at least a part of them will can be counted as real ad clicks (and that's why we have the click probability sliders on our interface, so that AdNauseam's behaviour is not as obvious to detect).

I hope that answers your question.
